app/src/main/java/org/mozilla/fenix/ext/BookmarkNode.kt

```diff
@@ -75,49 +51,69 @@ suspend fun BookmarkNode?.withOptionalDesktopFolders(
 
 private suspend fun virtualDesktopFolder(context: Context?): BookmarkNode? {
     val rootNode = context?.bookmarkStorage()?.getTree(BookmarkRoot.Root.id, false) ?: return null
-    return BookmarkNode(
-        type = rootNode.type,
-        guid = rootNode.guid,
-        parentGuid = rootNode.parentGuid,
-        position = rootNode.position,
-        title = rootTitles[rootNode.title],
-        url = rootNode.url,
-        children = rootNode.children
-    )
+    return rootNode.copy(title = rootTitles[rootNode.title])
```

Comment:
Is there a reason we weren't doing this form of copy originally?

Reply:
I did not write this code section, but the parts I did write used the Kotlin copy() operator for readability. The old, deep copy format here seems far too prone to typos and makes the code scan poorly.